missions/31_pony.txt

```diff
@@ -0,0 +1,158 @@
+quest_for_harmony_south_america = {
+
+	type = country
+	
+	category = MIL
+	
+	# target_provinces = {
+	#	region = south_america #to future me : pick 3 provinces in each continent, and when the player gets one of them fire an event (with good chance to win) to tell him if he succeded or not.
+	# } # EVENTUALLY UNUSED
+		
+	allow = {
+		is_at_war = no
+		num_of_ports = 1
+		religion_group = equestrian
+		has_country_flag = quest_for_harmony
+	}
+	success = {
+		has_country_flag = south_american_element # given via corresponding event
+	}
+	chance = {
+		factor = 1000
+	}
+	immediate = {
+		set_country_flag = quest_for_south_american_element
+		add_claim = 742 #Guayana
+		add_claim = 795 #Chichas
+		add_claim = 825 #Bogota (El Dorado !)
+		
+	}
+	abort_effect = {
+		remove_claim = 742 #Guayana
+		remove_claim = 795 #Chichas
+		remove_claim = 825 #Bogota
+	}
+	effect = {
+		add_prestige = 30
+		add_country_modifier = {
+			name = "south_american_element"
+			duration = -1
+		}
+	}
+}
+
+pony_united_lands = {
```

Comment:
Yeeeeaah, I suck at this sort of stuff. Fiddly language! I wanted this to be a decision, available once you have all the equestrian cultured provinces, but I couldn't get the allow to run a limit and only select the those cultured provinces. Sucks. So it became this instead.